Description: Patch sage-env
 Note that this file is only used during the package build and not installed.
 Instead the file debian/adhoc/sage-env is installed.
 .
 The LD_LIBRARY_PATH can probably be forwarded upstream, either way is fine.
Author: Tobias Hansen <thansen@debian.org>, Ximin Luo <infinity0@debian.org>

--- a/sage/src/bin/sage-env
+++ b/sage/src/bin/sage-env
@@ -264,12 +264,12 @@
 # depending on SAGE_ROOT and SAGE_LOCAL which are already defined.
 export SAGE_ETC="$SAGE_LOCAL/etc"
 export SAGE_SHARE="$SAGE_LOCAL/share"
-export SAGE_EXTCODE="$SAGE_SHARE/sage/ext"
+export SAGE_EXTCODE="$SAGE_SHARE/sagemath/ext"
 export SAGE_SPKG_INST="$SAGE_ROOT/local/var/lib/sage/installed"
 export SAGE_LOGS="$SAGE_ROOT/logs/pkgs"
 export SAGE_SRC="$SAGE_ROOT/src"
 export SAGE_DOC_SRC="$SAGE_SRC/doc"
-export SAGE_DOC="$SAGE_SHARE/doc/sage"
+export SAGE_DOC="$SAGE_SHARE/doc/sagemath"
 
 if [ -z "${SAGE_ORIG_PATH_SET}" ]; then 
     SAGE_ORIG_PATH=$PATH && export SAGE_ORIG_PATH 
@@ -325,12 +325,12 @@
 fi
 
 # For PARI/GP
-GP_DATA_DIR="$SAGE_LOCAL/share/pari" && export GP_DATA_DIR
-GPHELP="$SAGE_LOCAL/bin/gphelp" && export GPHELP
-GPDOCDIR="$SAGE_LOCAL/share/pari/doc" && export GPDOCDIR
+GP_DATA_DIR="/usr/share/pari" && export GP_DATA_DIR
+GPHELP="/usr/bin/gphelp" && export GPHELP
+GPDOCDIR="/usr/share/pari/doc" && export GPDOCDIR
 
-SINGULARPATH="$SAGE_LOCAL/share/singular" && export SINGULARPATH
-SINGULAR_EXECUTABLE="$SAGE_LOCAL/bin/Singular" && export SINGULAR_EXECUTABLE
+SINGULARPATH="/usr/share/singular" && export SINGULARPATH
+SINGULAR_EXECUTABLE="/usr/bin/Singular" && export SINGULAR_EXECUTABLE
 
 if [ -z "$SAGE_REPO_ANONYMOUS" ]; then
     SAGE_REPO_ANONYMOUS="git://trac.sagemath.org/sage.git"
@@ -420,8 +420,10 @@
     export PYTHONHOME
 fi
 
-LDFLAGS="-L$SAGE_LOCAL/lib -Wl,-rpath,$SAGE_LOCAL/lib $LDFLAGS"
+LDFLAGS="-L$SAGE_LOCAL/lib $LDFLAGS"
 export LDFLAGS
+LD_LIBRARY_PATH="$SAGE_LOCAL/lib:$LD_LIBRARY_PATH"
+export LD_LIBRARY_PATH
 
 # Use a matplotlib config directory specific to Sage and specific to
 # the version number of matplotlib, by setting the environment
@@ -468,7 +470,6 @@
 	fi
     fi
 fi
-export MAXIMA_PREFIX="$SAGE_LOCAL"
 
 PERL5LIB="$SAGE_LOCAL/lib/perl5:$PERL5LIB" && export PERL5LIB
 
@@ -642,9 +643,6 @@
     LD_LIBRARY_PATH="$SAGE_LOCAL/bin:$LD_LIBRARY_PATH" && export LD_LIBRARY_PATH
 fi
 
-# See trac 7186 -- this is needed if ecl is moved
-ECLDIR="$SAGE_LOCAL/lib/ecl/" && export ECLDIR
-
 # Handle parallel building/testing/...
 # See Trac Ticket #12016
 # First, figure out the right values for SAGE_NUM_THREADS (default
--- a/sage/build/make/install
+++ b/sage/build/make/install
@@ -11,7 +11,7 @@
 fi
 export SAGE_SRC="$SAGE_ROOT/src"
 export SAGE_SHARE="$SAGE_LOCAL/share"
-export SAGE_EXTCODE="$SAGE_SHARE/sage/ext"
+export SAGE_EXTCODE="$SAGE_SHARE/sagemath/ext"
 export SAGE_LOGS="$SAGE_ROOT/logs/pkgs"
 export SAGE_SPKG_INST="$SAGE_ROOT/local/var/lib/sage/installed"
 export SAGE_VERSION=`cat $SAGE_ROOT/VERSION.txt | sed 's+.*\ \(.*\),.*+\1+'`
--- a/sage/configure.ac
+++ b/sage/configure.ac
@@ -85,7 +85,7 @@
 fi
 SAGE_SRC="$SAGE_ROOT/src"
 SAGE_SHARE="$SAGE_LOCAL/share"
-SAGE_EXTCODE="$SAGE_SHARE/sage/ext"
+SAGE_EXTCODE="$SAGE_SHARE/sagemath/ext"
 SAGE_SPKG_INST="$SAGE_ROOT/local/var/lib/sage/installed"
 
 #---------------------------------------------------------
